backend/src/main/java/co/inventorsoft/scripty/service/EmailServiceImpl.java

```diff
@@ -51,9 +55,22 @@ public void sendEmailWithVerificationLink(final User user,
 
     public void resendEmailWithVerificationLink(final User user,
                                                 final VerificationToken token){
+        final String subject = "Confirmation Link";
+        MimeMessage mimeMessage = null;
+        try {
+            mimeMessage = constructMimeMessage(user, token.getToken(), prefixUrl, verificationEmailTemplate, subject);
+        } catch (MessagingException | IOException | TemplateException e){
+            e.printStackTrace();
```

Comment:
We should throw `ApplicationException` here, cause it doesn't make sense to proceed with `JavaMailSender#send` method if message is corrupted